docs/portfolios/Basics.md

```diff
@@ -0,0 +1,51 @@
+---
+tags: [Portfolios]
+---
+
+# Portfolio basics
+
+Portfolios are a container for an investor's cash and holdings. All investment activity on the WealthKernel platform (balances, orders, transactions) is directly tied to a portfolio.
```

Comment:
Is it worth adding a sentence something along the lines of "you can have multiple portfolios under a single account, allowing for different investment strategies, but note that we do not recommend mixing and matching mandate types."